### DISCUSSION TITLE
Trying to setup an extra centering command with custom size

### DISCUSSION THREAD

**Discussion by thesobercoder** [rxhanson/Rectangle#796] [Q&A]
Hi All,

I'm very new to Rectangle and Mac. Loving the software so far. Especially on my ultrawide screen, it is really useful. I'm trying to setup the extra centering command with a custom size following the docs, like this - 

```sh
defaults write com.knollsoft.Rectangle specified -dict-add keyCode -float 8 modifierFlags -float 1966080
defaults write com.knollsoft.Rectangle specifiedHeight -float 720
defaults write com.knollsoft.Rectangle specifiedWidth -float 1280
```

The problem is I'm not able to figure out what is the keyboard shortcut for this? Any help will be deeply appreciated. Thanks!

**Comment by rxhanson** [comment]
> This extra command is not available in the UI. You'll need to know which keycode and modifier flags you want (try the free key codes app: https://apps.apple.com/us/app/key-codes/id414568915)

Did you try that app? Alternatively you could use my KeyLimePie app and show the shortcut log: https://keylimepie.app

I don't know offhand what key code 8 and mod flags 1966080 are, but the idea is that you'd probably want to configure it yourself instead of taking whatever that command is (I wasn't the dev for that feature).

Even easier (but paid): the custom shortcut builder in Rectangle Pro: https://rectangleapp.com/pro
